python/GafferImageUI/ImageWriterUI.py

```diff
@@ -42,6 +42,22 @@
 import GafferImage
 from . import OpenColorIOTransformUI
 
+
+layoutVariableDesc = """
+Special context variables available for setting layout plugs:
+
+* ${imageWriter:viewName} : The current view
```

Comment:
Probably worth using backticks to get the nice code formatting applied to `${name}` in the docs.